src/resolvers/Query/getAllAgendaCategories.ts

```diff
@@ -0,0 +1,23 @@
+import type {
+  QueryResolvers,
+  AgendaCategory,
+} from "../../types/generatedGraphQLTypes";
+import { AgendaCategoryModel } from "../../models";
+import { errors, requestContext } from "../../libraries";
+import { AGENDA_CATEGORY_NOT_FOUND_ERROR } from "../../constants";
+/* eslint-disable */
```

Comment:
Why this comment?

Reply:
Hello, it is giving me errors for TSdocs I have researched a lot, but i do not see a reason why the lint errors are coming. The code and doc style seems correct to me. After a lot of trying, i finally added this comment.